src/components/Box/BoxCoreFunctions.scss

```diff
@@ -0,0 +1,45 @@
+@use "sass:string";
+@use "sass:list";
+
+// BOX core functions: mock functions
+// These function should be imported from "monday-ui-styles"
+
+// Capitalize
+// - Helper function
+// @arguments: $string
+// @return capitalize string
+@function capitalize($string) {
+  @return to-upper-case(string.slice($string, 1, 1)) + string.slice($string, 2);
+}
+
+// Contain
+// - Helper function
+// @arguments [list] $list
+// @arguments [$value] $value
+// Return whether `$value` is contained in `$list` , Boolean
+@function contain($list, $value) {
```

Comment:
I think this function is more general and maybe we can pust them in more general file which not only related to box?

Reply:
like general css functions file

Reply:
it should be a general function, its here as mock.